app/javascript/components/layout/SharedLayout.jsx

```diff
@@ -3,12 +3,10 @@ import PropTypes from 'prop-types';
 import Header from 'components/layout/Header';
 import Footer from 'components/layout/Footer';
 
-import 'stylesheets/layout';
-
 const SharedLayout = ({ children }) => (
-    <div className="layout">
+    <div className="flex flex-col h-full">
```

Comment:
Deleted a CSS file and moved the styles back in here since there aren't that many of them.